### PR TITLE
Finished the Newsweek

### PR DESCRIPTION
**Pull request by emasdev**
Finished the Frontpage of Newsweek page using almost bootstrap framework.
It has a header, a main news section, some grids, newsletter section, and footer.

Everything is responsive

### CODE REVIEW

**Comment by rammazzoti2000** [comment]
### Pull Request under the review by Alexandru Bangau :raised_hands:

File: style.css [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  padding: 0.1em 0.5em;
+  font-size: 0.8rem;
+}
+
+.navbar-logo {
+  width: 140px;
+}
+
+.vs-border-red {
+  border-top: 4px solid #f92859 !important;
```
Comment (rammazzoti2000) [inline style.css]: You should not use `!important` property

File: style.css [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+}
+
+.vs-border-blue {
+  border-top: 4px solid #1c87fb !important;
```
Comment (rammazzoti2000) [inline style.css]: You should not use `!important` property

**Review by rammazzoti2000 — CHANGES_REQUESTED** [review]
### Hello @emasdev I am Alex, your code reviewer for this project :raised_hands: 
Your project looks amazing, you made great use of HTML semantic, and is impressive how much you used bootstrap.
Your readme is descriptive, the pull request has meaningful title and description, the repo overall looks professional, I just wonder how come so few commits for such a big project :question: 
Anyway, there are some issues that are preventing me from approving this project
### :stop_sign:  Required Changes :construction: :construction_worker: 
- [x] The HTML code is not passing validation, it should pass validation using this recommended [validator](https://validator.w3.org/#validate_by_input)  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/validation_requirement.png) from Pathwright)
image
- [x] The title for each section should be a bit bolder, it seems they are quite light with respect to the original
- [x] The links section in the navbar should stretch to the whole width
#### Clone:
image
#### Original:
image
- [x] On mobile screen the Featured Stories switches with the Top Stories, the first section from the top should be Top Stories
- [x] The social buttons in the footer when on medium screens are not properly aligned
#### Clone:
image
#### Original:
image
- [x] On mobile screens the page is not displaying properly, including the navbar, you can see that just under 500px from the screenshot bellow
image
####
### <em>Kindly fix the REQUIRED issues and send another code review request.

Cheers and Happy coding! :clap: 

##### Code review by Alexandru Bangau

(Feel free to contact me on slack by the handle of alexandru if you have any additional question)</em>

### :stop_sign:  Status: Changes required.

**Review by rammazzoti2000 — CHANGES_REQUESTED** [review]
#### :stop_sign: Status: Required Changes

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @emasdev, great job :1st_place_medal:  The project looks great :+1: However, let us improve the project and make it even better. :smile: 
- Kindly address the comments by the previous TSE for which he talked about content not displaying properly on-screen < 500px on ( mobile device), including the subscribe button and the login
```Currently```
c
```Original```
o

[OPTIONAL]
Kindly follow this order on mobile device => Top story, Featured stories, Opinion, More stories, Heroes of the pandemic, Latest News, In the magazine

Kindly address this issue and submit another code review :smile: 
_Happy coding_ :man_technologist: 
Daniel Larbi Addo on slack

**Review by serdg0 — APPROVED** [review]
**Congratulations** @emasdev  :fireworks: :100: :fireworks:
Feel free to merge.
Keep the good work in your next project!
**Happy Coding!** :computer: :nerd_face:
**_Sergio Diaz_**

**Pull request merged**